gettsim/tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+import yaml
+
+from gettsim.config import ROOT_DIR
+
+
+@pytest.fixture(scope="session")
+def tax_policy_data():
+    return yaml.safe_load(open(f"{ROOT_DIR}/data/param.yaml", "rb"))
```

Comment:
``yaml.safe_load((ROOT_DIR / "data" / "param.yaml").read_text())``

Reply:
See https://docs.python.org/3/library/pathlib.html for information on how pathlib works. It is amazing :)

Reply:
Changed accordingly. We should right down somewhere that we want paths that way. I changed some in the beginning from this format to f-strings.